public/manifest.json

```diff
@@ -13,6 +13,28 @@
     "default_popup": "index.html",
     "default_icon": "images/default.png"
   },
+  "background": {
+    "scripts": [
+      "background.js"
```

Comment:
Is this supposed to be `background.js`? If so I believe it is the emitted result of `background.ts`.

Reply:
Good point. I haven't tried `.ts`, so I just did and got: `Could not load background script 'background.ts'.`
And Yes to the second sentence.